crypto/blake2/blake2b.c

```diff
@@ -252,18 +303,28 @@ int BLAKE2b_Update(BLAKE2B_CTX *c, const void *data, size_t datalen)
  */
 int BLAKE2b_Final(unsigned char *md, BLAKE2B_CTX *c)
 {
+    uint8_t outbuffer[BLAKE2B_OUTBYTES] = {0};
+    uint8_t *target = outbuffer;
     int i;
 
+    /* Avoid writing to the temporary buffer if possible */
+    if (c->outlen == BLAKE2B_DIGEST_LENGTH) {
+        target = md;
+    }
```

Comment:
No `{}` for a single line if

Reply:
Does this also apply to for loops such as the actual hash copy?